process-migration-service/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence version="2.1"
+	xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
+	<persistence-unit name="migration-unit"
+		transaction-type="JTA">
+		<provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+		<!--it seems defines the jta-data-source or not it still works, so comment 
+			it out for now <jta-data-source>java:jboss/datasources/ExampleDS</jta-data-source> -->
```

Comment:
this could be due to default data source for jpa configuration but I guess we should support custom data sources to be defined outside for other db so having more meaningful data source name than ExampleDS might be useful

Reply:
It can indeed be done in the configuration file, see the examples. I will remove this comment.